.github/workflows/release-operator-container.yml

```diff
@@ -11,46 +11,78 @@ env:
+      - uses: actions/setup-java@v3
         with:
-          images: quay.io/keycloak/keycloak-operator
-          tags: |
-            type=ref,event=branch
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
+          distribution: 'temurin'
+          java-version: ${{ env.JDK_VERSION }}
+          cache: 'maven'
+
+      - name: Build locally Keycloak artifacts to avoid depending on Maven
+        run: |
+          mvn -P operator -pl operator -am clean install -DskipTests     
 
       - name: Login to Quay
         uses: docker/login-action@v2
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_TOKEN }}
-
-      - uses: actions/setup-java@v3
+          
+      - name: Container metadata from Quay
+        id: meta_quay
+        uses: docker/metadata-action@v4
         with:
-          distribution: 'temurin'
-          java-version: ${{ env.JDK_VERSION }}
-          cache: 'maven'
-
-      - name: Build locally Keycloak artifacts to avoid depending on Maven
+          images: abstractj/keycloak-operator
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+                      
+      - name: Build and push the operator image to Quay
+        working-directory: operator
         run: |
-          mvn -P operator -pl operator -am clean install -DskipTests
+          echo "${{ steps.meta_quay.outputs.tags }}" | xargs -I {} \
+          mvn clean package \
+              -DskipTests \
+              -Dquarkus.container-image.username=${{ secrets.QUAY_USERNAME }} \
+              -Dquarkus.container-image.password=${{ secrets.QUAY_TOKEN }} \
+              -Dquarkus.container-image.image={} \
+              -Dquarkus.container-image.build=true \
+              -Dquarkus.container-image.push=true \
+              -Dquarkus.container-image.registry=quay.io
+      
+      - name: Login to Docker
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+          logout: true
+
+      - name: Container metadata from Docker
+        id: meta_docker
+        uses: docker/metadata-action@v4
+        with:
+          images: keycloak/keycloak-operator
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
 
-      - name: Build the operator Docker image
+      - name: Build and push the operator image to Docker
         working-directory: operator
         run: |
-          echo "${{ steps.meta.outputs.tags }}" | xargs -I {} \
+          echo "${{ steps.meta_docker.outputs.tags }}" | xargs -I {} \
           mvn clean package \
               -DskipTests \
+              -Dquarkus.container-image.username=${{ secrets.DOCKER_USERNAME }} \
```

Comment:
those credentials, "should" not be needed after a successful login.

Reply:
@andreaTP I had the chance to experiment with that, before submitting this PR. For some reason, even after a successful login, I was getting HTTP 401. To be honest, I had no clue why. That was the reason why I explicitly included username/password here.

Reply:
@abstractj I can see that you removed this change, have you managed to get it to work?

Reply:
@andreaTP yes, all the changes were tested on my dummy organization on GH, DockerHub and Quay.io.